elasticai/creator/nn/fixed_point/mac/layer.py

```diff
@@ -4,7 +4,7 @@
 from elasticai.creator.nn.fixed_point.two_complement_fixed_point_config import (
     FixedPointConfig,
 )
-from elasticai.creator.vhdl.simulated_layer import SimulatedLayer
+from elasticai.creator.testing.simulated_layer import SimulatedLayer
```

Comment:
Same case here as with the other comment. I guess i added `elasticai.creator.vhdl.simulated_layer` as an exception previously. We could do the same in this case, but i think i'd prefer to move the simulated layer to the top-level tests package.
Especially as the whole simulated layer module is not supposed to stay.